e2e-tests/test_runner.sh

```diff
@@ -107,14 +121,15 @@ echo -e "┌──────────────────────
 echo -e "│                              ${TXT_BIYLW}TEST SUMMARY${TXT_NORML}                              │"
 echo -e "├────────────────────────────────────────────────────────────────────────┤"
 printf  "│ ${TXT_BIBLK} Total tests executed:  ${TXT_BIBLU} %3d ${TXT_NORML}                                          │\n" "${TOT_EXEC_TESTS}"
+PASSED_TESTS=$((TOT_EXEC_TESTS - TOT_FAIL_TESTS))
+printf  "│ ${TXT_BIBLK} Passed tests:          ${TXT_BIGRN} %3d ${TXT_NORML}                                          │\n" "${PASSED_TESTS}"
+printf  "│ ${TXT_BIBLK} Failed tests:          ${TXT_BIRED} %3d ${TXT_NORML}                                          │\n" "${TOT_FAIL_TESTS}"
+
 if [ ${TOT_FAIL_TESTS} -ne 0 ]; then
     echo -e "├────────────────────────────────────────────────────────────────────────┤"
-    printf  "│ ${TXT_BIBLK} Failed tests:          ${TXT_BIRED} %3d ${TXT_NORML}                                          │\n" "${TOT_FAIL_TESTS}"
     for failed_test in "${FAILED_TESTS[@]}"; do
         printf "│     - %-64s │\n" "${failed_test}"
     done
-    echo -e "└────────────────────────────────────────────────────────────────────────┘"
-    exit 1
```

Comment:
I guess that there's a little undesirable side effect by remove these lines: if some tests fails the script not fail.
That could be an issue both for the developer that cannot leverage on script output to understand in the some test failed and for the CI.

Ok, removing the footer duplication could be nice but we should define an exit value variable to recover the right behavior.

Reply:
Thanks both, i've added the exit status back in.

Reply:
@drgora : @JackPiri already open a PR to fix it (I've raised a red flag yesterday when I saw that Daniele's PR fixed some format issues on my last PR.

Reply:
Great for the other PR, thank you.